packages/rendermime/src/renderers.ts

```diff
@@ -414,6 +423,11 @@ export namespace renderMarkdown {
      */
     host: HTMLElement;
 
+    /**
+     * The `id` of the element should be scrolled into view.
+     */
+    fragment: string | null;
```

Comment:
Can we make this an optional parameter instead of `string | null`?

Reply:
I think the `fragment` parameter can be moved into the `renderedHTML` options, since they are both dealing with rendered DOM content. Then we can use fragments for both html content and markdown content.

Reply:
> Can we make this an optional parameter instead of `string | null`?

Yes. I made it `string | null` for coherence with the other optional interface members.

> I think the `fragment` parameter can be moved into the `renderedHTML` options

It would be an excellent idea if `renderMarkdown` used `renderHTML` to put the HTML code into the host element. But, if I'm right, [it does not][1] at the moment.

[1]: https://github.com/jupyterlab/jupyterlab/blob/ff47903e8adfc4cd13a688734355e18b48fb2dbe/packages/rendermime/src/renderers.ts#L355